sdk/translation/ai-translation-text/samples-dev/breakSentence.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary This sample demonstrates how to make a simple call to the Azure Text Translator service to get sentences' boundaries.
+ */
+import TextTranslationFactory, { ErrorResponseOutput, TranslatorCredential, InputTextItem, FindSentenceBoundariesQueryParamProperties, BreakSentenceItemOutput } from "@azure-rest/ai-translation-text";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint = process.env["ENDPOINT"] || "https://api.cognitive.microsofttranslator.com";
+const apiKey = process.env["TEXT_TRANSLATOR_API_KEY"] || "<api key>";
+const region = process.env["TEXT_TRANSLATOR_REGION"] || "<region>";
+
+export async function main() {
+  console.log("== Get Sentence Boundaries sample ==");
+
+  const translateCedential = new TranslatorCredential(apiKey, region);
+  const translationClient = TextTranslationFactory(endpoint, translateCedential, undefined);
+
+  const inputText: InputTextItem[] = [
+    { text: "zhè shì gè cè shì。" }
+  ];
+  const parameters: FindSentenceBoundariesQueryParamProperties & Record<string, unknown> = {
+    language: "zh-Hans",
+    script: "Latn"
+  };
+  const breakSentenceResponse = await translationClient.path("/breaksentence").post({
+    body: inputText,
+    queryParameters: parameters
+  })
+
+  if (breakSentenceResponse.status !== "200") {
```

Comment:
I think we should use `isUnexpected` in the samples too.

Reply:
Done